.github/styles/config/vocabularies/default/accept.txt

```diff
@@ -8,48 +52,41 @@ Acala
 Alfajores
 Andromeda
 Apothem
-Arbitrum
 Arianee
 Astar
-Aurora
 Auth Kit
 Autobahn
 Avax
 BNB
-Binance
-Blockchain
 Blocknative
 Blockscout
 Boba
 Bobabeam
+Bundlers
 C-Chain
 C1
 CORS
-CRA
 Canto
+Callout
```

Comment:
This is the name of a React component. Does this mean we will need to add all the names of future components here. As we will add widgets and probably new layouts, this will be a big overhead I think

Reply:
Same here